smart_campus/app/forms.py

```diff
@@ -19,3 +19,15 @@ def __init__(self, *args, **kwargs):
     class Meta:
         model = Station
         fields = ('name', 'content', 'category')
+
+
+class CategoryForm(forms.ModelForm):
+    class Meta:
+        model = StationCategory
+        fields = '__all__'
+
+
+class PartialRewardForm(forms.ModelForm):
```

Comment:
What is this form for?

Reply:
its used for the post a partial reward form whose excluded stations fields of reward model

ref: [Django PartialForm](https://docs.djangoproject.com/en/1.11/topics/forms/modelforms/#selecting-the-fields-to-use)